src/hotspot/share/memory/metaspace/binList.hpp

```diff
@@ -191,10 +192,14 @@ class BinListImpl {
     for (int i = 0; i < num_lists; i++) {
       const size_t s = word_size_for_index(i);
       int pos = 0;
+      Block* b_last = nullptr; // catch simple circularities
       for (Block* b = _blocks[i]; b != nullptr; b = b->_next, pos++) {
         assert(check_canary(b, s), "");
+        assert(b != b_last, "Circle");
         local_counter.add(s);
+        b_last = b;
       }
+      if (UseNewCode)printf("\n");
```

Comment:
I guess this line is a leftover to be removed?

Reply:
Yep thanks for spotting

Reply:
So that was causing the empty lines in my logs (facepalm)